index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<head>
+    <link href="assets/css/lib/bootstrap.min.css" rel="stylesheet" type="text/css"/>
+    <link href="assets/css/index.css" rel="stylesheet" type="text/css"/>
+
+    <script src="assets/js/lib/jquery-1.11.2.min.js"></script>
+    <script src="assets/js/lib/bootstrap.min.js"></script>
+    <script src="assets/js/index.js"></script>    
+</head>
+
+<body>
+
+
+<div id="ProjectView" class="container">
+
+	<div id="NameView" class="row">
+		my_chair
+	</div>
+
+	<div id="HistoryView" class="row">
+		history
+	</div>
+
+	<div id="EaselView" class="row">
+		<div id="ToolbarView" class="col-md-4">
+			tools
+		</div>
+
+		<div id="PadView" class="col-md-4">
+			pad
+		</div>
+	</div>
+
+	<div id="SuggestionsView" class="row">
```

Comment:
There was discussion as to whether suggestions should be a row below both Toolbar and Pad, or only below Toolbar. Should be easy to change, so just making a note for now.